tests/bin/test_tmux_layout.py

```
@@ -1,301 +1,301 @@
+        # tmuxセッションを作成
+        busd.ensure_session()
+        time.sleep(0.5)
+        
+        # 子エージェント用のディレクトリを作成
+        for i in range(1, 3):
+            task_dir = tmppath / "work" / f"T00{i}"
+            task_dir.mkdir(parents=True)
+        
+        # 最初の子エージェントを起動
+        pane1 = busd.spawn_child("T001", "work/T001", "frames/impl/CLAUDE.md", "Task 1")
+        assert pane1 is not None, "Failed to spawn first child"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)